### PR TITLE
Related articles

### PR DESCRIPTION
**Pull request by manggalang**

### CODE REVIEW

File: app/assets/stylesheets/pages/articles/article-detail/_related-articles.scss [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+		background-image: image-url("bg.jpg");
+		background-size: cover;
+		background-position: center;
+		height: 179px;
+		border-radius: 15px;
+		transition: all .3s ease-in-out;
+	}
+
+	&__title{
+		margin: 0;
```
Comment (Fajaru1503) [inline app/assets/stylesheets/pages/articles/article-detail/_related-articles.scss]: i think you need more gap here, check the mockup

File: app/views/client/articles/_related-articles.html.erb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<div class="related-articles">
+	<div class="container-large related-articles__item-wrapper">
+
+		<div class="related-articles__item">
```
Comment (Fajaru1503) [inline app/views/client/articles/_related-articles.html.erb]: use tag a

File: app/assets/stylesheets/pages/articles/article-detail/_related-articles.scss [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+	    @media only screen and (max-width: 751px){
+	    	font-size: 14px;
+	    }
+	}
+
+	&__author{
+		margin: 0;
+		font-size: 12px;
+		color: $textColorSecondary;
+		line-height: 2.6;
```
Comment (Fajaru1503) [inline app/assets/stylesheets/pages/articles/article-detail/_related-articles.scss]: don't change the line-height, use margin

File: app/assets/stylesheets/pages/articles/article-detail/_related-articles.scss (hunk 1 above)
Comment (manggalang) [reply app/assets/stylesheets/pages/articles/article-detail/_related-articles.scss]: sorry im not focused

File: app/views/client/articles/_related-articles.html.erb (hunk 2 above)
Comment (manggalang) [reply app/views/client/articles/_related-articles.html.erb]: already to change

File: app/assets/stylesheets/pages/articles/article-detail/_related-articles.scss (hunk 3 above)
Comment (manggalang) [reply app/assets/stylesheets/pages/articles/article-detail/_related-articles.scss]: already to change with margin

**Pull request merged**